tests/subsys/mem_mgmt/mem_attr_heap/boards/qemu_cortex_m3.overlay

```diff
@@ -0,0 +1,45 @@
+#include <common/mem.h>
+#include <zephyr/dt-bindings/memory-attr/memory-attr.h>
+#include <zephyr/dt-bindings/memory-attr/memory-attr-sw.h>
+
+/ {
+	mem_cache: memory@20008000 {
```

Comment:
nit: `mem_cache` as a name misled me, I thought it was a region serving as a cache at first glance, actually it is a region with catchable attrs. I am not sure if this is an issue for others, if not, just ignore this:)